dotnet/src/AutoGen.SourceGenerator/AutoGen.SourceGenerator.csproj

```diff
@@ -9,7 +9,9 @@
     <UserSecretsId>35954224-b94e-4024-b0ef-7ba7cf80c0d8</UserSecretsId>
 	  <GetTargetPathDependsOn>$(GetTargetPathDependsOn);GetDependencyTargetPaths</GetTargetPathDependsOn>
 	  <LaunchDebugger>false</LaunchDebugger>
+	  <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```

Comment:
Remove this? The RS1036 is triggered when you didn't set `EnforceExtendedAnalyzerRules` to true. Since you already suppress that warning, you no longer need to re-set `EnforceExtendedAnalyzerRules` to true